src/Microsoft.NetCore.Analyzers/UnitTests/Tasks/DoNotCreateTasksWithoutPassingATaskSchedulerTests.cs

```diff
@@ -70,48 +57,44 @@ public void M6(object obj)
 }
 ");
 
-            VerifyBasic(@"
+            await VerifyVB.VerifyAnalyzerAsync(@"
 Imports System
 Imports System.Threading
 Imports System.Threading.Tasks
 
 Class C
-	Public Sub M(task As Task, factory As TaskFactory, scheduler As TaskScheduler, continuationOptions As TaskContinuationOptions, creationOptions As TaskCreationOptions, ct As CancellationToken)
-		task.ContinueWith(AddressOf M2, scheduler)
-		task.ContinueWith(AddressOf M3, Nothing, scheduler)
-		task.ContinueWith(AddressOf M2, ct, continuationOptions, scheduler)
-		task.ContinueWith(AddressOf M3, Nothing, ct, continuationOptions, scheduler)
+    Public Sub M(task As Task, factory As TaskFactory, scheduler As TaskScheduler, continuationOptions As TaskContinuationOptions, creationOptions As TaskCreationOptions, ct As CancellationToken)
+        task.ContinueWith(AddressOf M2, scheduler)
+        task.ContinueWith(AddressOf M3, Nothing, scheduler)
+        task.ContinueWith(AddressOf M2, ct, continuationOptions, scheduler)
+        task.ContinueWith(AddressOf M3, Nothing, ct, continuationOptions, scheduler)
 
-		factory.StartNew(AddressOf M5, ct, creationOptions, scheduler)
-		factory.StartNew(AddressOf M6, Nothing, ct, creationOptions, scheduler)
-	End Sub
+        factory.StartNew(AddressOf M5, ct, creationOptions, scheduler)
+        factory.StartNew(AddressOf M6, Nothing, ct, creationOptions, scheduler)
+    End Sub
 
-	Public Sub M2(task As Task)
-	End Sub
+    Public Sub M2(task As Task)
+    End Sub
 
-	Public Sub M3(task As Task, obj As Object)
-	End Sub
+    Public Sub M3(task As Task, obj As Object)
+    End Sub
 
-	Public Sub M4(Of TResult)(task As Task, obj As TResult)
-	End Sub
+    Public Sub M4(Of TResult)(task As Task, obj As TResult)
+    End Sub
 
-	Public Sub M5()
-	End Sub
+    Public Sub M5()
+    End Sub
 
-	Public Sub M6(obj As Object)
-	End Sub
+    Public Sub M6(obj As Object)
+    End Sub
```

Comment:
⚠️ All of this indentation should not have changed. The indentation is located within string literals. Applies to multiple files.